tests/test_coadd_outputs.py

```diff
@@ -143,8 +143,8 @@ def test_coadd_transmission_curves(self):
 
             to_check = ctx.choice(len(cat), size=n_object_test, replace=False)
             for index in to_check:
-                coadd_coord = geom.SpherePoint(cat["coord_ra"].values[index]*geom.degrees,
-                                               cat["coord_dec"].values[index]*geom.degrees)
+                coadd_coord = geom.SpherePoint(cat["coord_ra"][index]*geom.degrees,
+                                               cat["coord_dec"][index]*geom.degrees)
```

Comment:
This is all fine, but the flexible thing to do is `np.asarray(cat["coord_ra"])[index]` which will work with any input that supports numpy `__array__` access without doing a data copy.  (including pandas, astropy, etc)